books/free-programming-books-langs.md

```diff
@@ -1135,6 +1135,7 @@ Books on general-purpose programming that don't focus on a specific language are
 
 * [Airbnb JavaScript Style Guide](https://airbnb.io/javascript/) - Airbnb (HTML)
 * [Basic JavaScript for the impatient programmer](http://www.2ality.com/2013/06/basic-javascript.html) - Axel Rauschmayer (HTML)
+* [Bible of JS](https://shorturl.at/ax056) - Sheryians Coding School (JavaScript)
```

Comment:
we don't accept shortened links

Reply:
'https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbElBUUg3MlJqMTd0Sk90azliZ1hKUDBKdUs1UXxBQ3Jtc0tuSWZOcHhJeVRuamJob2Vfamc2UU5LN09KOXFlQjlVRjk0ejc1bmZIN1ozVEFXcGZYWDZGbjJUTlFRbjQ3dUM2NlJsNGF5SWw4WmhXbU01TXVGaEVkVFFNVEhSTnJTVUVBaFFiaTMydUxLSjY4WFFmQQ&q=https%3A%2F%2Fsheryians.com%2Fdownload%2Fbibleofjs_by_sheryians&v=Aw_SmcpEOdY'      So should i add this link? Is this valid?

Reply:
It seems the link you provided redirects to a different site. We don't accept redirected links; we prefer original sources. Share the direct source for consideration.

Reply:
Finally I got the original url. I've updated it check it out